relay-general/src/store/transactions/processor.rs

```diff
@@ -1,6 +1,7 @@
 use std::borrow::Cow;
 
 use relay_common::SpanStatus;
+use relay_metrics::is_valid_metric_name;
```

Comment:
is it bad to import another crate just for this small function?

Reply:
Better to move that function to `relay-common` IMO.